src/queries/GetTRBRequestRelatedRequestsQuery.ts

```diff
@@ -0,0 +1,29 @@
+import { gql } from '@apollo/client';
+
+const GetTRBRequestRelatedRequestsQuery = gql`
```

Comment:
Can you check the form of newer GQL calls?

I'm not caught up with it myself. There is newer work @patrickseguraoddball @aterstriep are doing
https://github.com/CMS-Enterprise/easi-app/pull/2744/files#diff-2284b8bb876578574fb0ced8cb78a7b6410abc7555d71dfc7c25fb873c002af7R5

Also looks like this new file is in the old folder.

Reply:
ok - think i got this right. thanks Adam!